api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -97,6 +98,11 @@ def _get_venues(form: forms.GetVenuesListForm) -> list[offerers_models.Venue]:
     if form.offerer.data:
         base_query = base_query.filter(offerers_models.Venue.managingOffererId.in_(form.offerer.data))
 
+    if form.only_validated_offerers.data:
+        base_query = base_query.join(offerers_models.Venue.managingOfferer).filter(
+            offerers_models.Offerer.validationStatus == ValidationStatus.VALIDATED
```

Comment:
Détail : on utilise plutôt l'_hybrid_property_ qui facilite la lecture : 
```suggestion
            offerers_models.Offerer.isValidated
```

Reply:
J'avais essayé, mais je pensais que ça ne marchait pas. C'est modifié :)